### PR TITLE
Unknown shuttle event changes

### LINKED ISSUE #27972: The Unknown shuttles are a admin nightmare
## Description
The amount of Ahelps these events cause is quite frankly insane and could be made better by fixing these few issues
Ill run through the issues quickly

Syndie Evac Pod: they are not antags despite being employed by the syndicate so they can't take any action against the station, and if they do its selfantag.

Cargo shuttle: Can spawn with any crates leading to some ungodly loot the moment they spawn, such as guns and there is even a chance for a free throngler

Chef shuttle: Spawns with carp toxin that could be used to just round remove people if used correctly

CLOWN SHUTTLE: spawns with 3 non-coged ghost role clown spiders, the shuttle also spawns with things like dehydrated space carps

Evac pod: this one is fine tbh

all of the problems with them could be solved if the humans were made antags with the sole objective to survive via evac
and the clown spiders either being replaced OR being coged and renamed to something that would tell the players they are friendly.
on top of other issues such as them not having IDs, now there is an RP argument for it, but they should just come with IDs of their job but with no access or just the PDA at least

### PR DESCRIPTION
**Pull request by Flareguy**
## About the PR
Makes major adjustments to many parts of the unknown shuttle event.
Fixes #27972.

The Travelling Chef role is now a "neutral" antagonist, like the closet skeleton.
The cargo shuttle & pod events have been replaced with a ship pilot role. This role is also a neutral antagonist.
The syndicate evac pod event has been retooled in to the Syndicate Raider event. Like the others, they are neutral antagonists.

The clown shuttle has been removed since #28053 is going to remove it anyways. It apparently wasn't very conductive to roleplay, even on MRP.

I understand that this may seem too far for the freeze, but the approaching shuttle event was merged a day before the freeze, so there wasn't really any chance to fix it. The changes here should not require too much tuning other then just reducing the event weights, anyways.

## Why / Balance
Even ignoring the massive admin issues these roles cause (look at the linked issue,) of the unknown shuttles (with the exception of the travelling chef) are just conceptually very odd ghost roles. Almost all of these collectively suffer from being nearly the same as crew roles, and having little directive or agency to influence the round due to their lack of objectives & non-antag status.

Take the cargo shuttle for instance. There is no goal or driving factor other then "you are a cargo tech." It is basically just a random cargo tech respawn, except you probably didn't even come in with a plan like you would if you would have spawned as a cargo tech roundstart. It is likely that anyone picking this role will eventually SSD due to not knowing what to do / getting bored, or just not pick it due to knowing they're not going to have fun.

By making all of these roles able to have full agency to do whatever they want, they should be able to drive the round in whatever fashion they please, and eliminate the admin issues completely.

The reduced amount of shuttle events (6 > 3) *should* make the spawn rates for these less overwhelming, though I obviously have no idea if they are still overwhelming.

## Media
Content Client_LvG6tZZRpa
Poor screenshot of the one new ship (the shuttle pilot shuttle.)

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
🆑 
- tweak: All unknown shuttle ghost roles are now "neutral" like closet skeletons, and have no allegiance.
- tweak: Replaced the cargo tech & evacuation pod events with a new event: the shuttle pilot.

### CODE REVIEW

**Comment by Dutch-VanDerLinde** [comment]
Changelog broken no CL

**Review by TheShuEd — APPROVED** [review]

**Comment by Djungelskog2** [comment]
ok but itd be really weird if they didnt have to follow escalation

**Comment by Djungelskog2** [comment]
more station friendly personnel ghost roles would be fun
My suggestions is keeping the cargo and travelling chefs the same but increasing their odds of being traitors drastically, ive never found issue with "Just being crew", although you may argue that the chef isnt part station or its crew he should act as a visitor at the very least. I like the reworking of the regular evac pod into a neutral pilot, but they should also follow escalation. Lastly the syndi raider event, im not sure how it works but im guessing its just an even more overwhelming sleeper agent event in functionality which is a pretty large issue in my opinion, Theres 100% an issue with the syndi shuttle at the moment but there are more creative and stand-out antag shuttle ideas that could be used, a ship overtaken by some anomaly, a skeleton pirate who has large amounts of loot from plundering another station and is an OUTRIGHT antag with the goal of stealing shit or a neutral zookeeper carrying large amounts of feral tarantulas. The syndi raid seems like its based on loads of already existing midrounds without adding much unique experience or round variety other than more for sec to handle, it could be done in a way to promote RP, funny unique scenarios or having small standalone interventions and interactions with the new shuttles rather than just "more syndis"

**Comment by Djungelskog2** [comment]
Comparing every neutral antag to the skeleton is a really bad idea because they are free to do anything they like, complete NRP and murderboning and sabotage. They are a primordial force of chaos and a normal humanoid being shouldnt be on that standard unless you are outright an antagonist. More Neutral roles are a fun concept but by comparing them all to a skeleton all it boils down to is "NRP, Crime, Murder" which is what makes the skeleton scary and unique and more roles having those rules will just lead to less fun of the crews end

**Comment by thebadman4662** [comment]
At least syndie crew gets to act like enemies they are now.

**Comment by Savsj** [comment]
Why are we just removing an amazing ghost role for RP in favor of more of the same antag we get constantly anyway? Like the roles from the shuttles are already station neutral, why are making them antags when tbh theyre better as non antags? This feels like a notable downgrade for RP potential.

**Comment by IProduceWidgets** [comment]
I have some reservations about making them all neutral- I think we should have some friendly ghost roles.

My bigger issue though is I think this is a bandaid for a lack of ways to convey neutral or friendly status/goals to players. We desperately need a way to put objectives or at least indicators in the C character menu. As well as a way to convey this to admins. Neutral roles don't even show up to admins as special but are antags in all but name.

A side note, and this could be another PR, is it could be worthwhile to just make prototype roles for shuttle events with differing goals / alignment. Those could be useful for admemes, and it could be interesting to have versions of these events where the crew that shows up is hostile when usually they are friendly.

**Comment by Killerqu00** [comment]
Please do not make them neutral antagonists. Most ghost roles are already not enjoyable due to being extremely disposable, for example, rat kings being killed on sight without any interactions whatsoever. These shuttle roles have a chance to actually be interesting for roleplay if done right, and this is not what will improve them.

**Comment by Cojoke-dot** [comment]
Making them neutral gives a much more interesting interaction with sec and command. Can these people from space be trusted? Can I allow this syndicate agent to walk free or do I just perma them? Much more interesting than "huh, some people in weird clothing, go get a job or something, I don't care"

**Comment by Savsj** [comment]
Making them neutral is just going to make them valid, they are already an interesting interaction with command and crew by saying "the people with cool loot i want are free game because they are not crew so i am allowed to butcher them in a crowded hallway" i believe giving them a chance to be antaggonists and actual syndies like with an uplink would be neat

**Comment by Cojoke-dot** [comment]
> Making them neutral is just going to make them valid, they are already an interesting interaction with command and crew by saying "the people with cool loot i want are free game because they are not crew so i am allowed to butcher them in a crowded hallway" i believe giving them a chance to be antaggonists and actual syndies like with an uplink would be neat

This is where the "don't be a dick rule" comes in. Either way sec is not going to like someone just murdering a non-skeleton and it's not hard to get in the manifest.

**Comment by Killerqu00** [comment]
> > Making them neutral is just going to make them valid, they are already an interesting interaction with command and crew by saying "the people with cool loot i want are free game because they are not crew so i am allowed to butcher them in a crowded hallway" i believe giving them a chance to be antaggonists and actual syndies like with an uplink would be neat
> 
> This is where the "don't be a dick rule" comes in. Either way sec is not going to like someone just murdering a non-skeleton and it's not hard to get in the manifest.

It does not apply to neutral antags, you can kill them on the spot without consequences

**Comment by Cojoke-dot** [comment]
Sure, but does that happen... uh... almost ever? I've been skeleton tons of times and have not once had that happen. The only time I've seen that happen is in the Armok video and that's a unique situation.

**Comment by SlamBamActionman** [comment]
I see a missed opportunity to make the Cargo techs a neutral antag affiliated with a non-NT company (e.g. DonkCo). 

> Sure, but does that happen... uh... almost ever? I've been skeleton tons of times and have not once had that happen. The only time I've seen that happen is in the Armok video and that's a unique situation.

This def happens. Not often, as people generally see skeletons as a fun role to have on the station, but I've seen it happen and even enforced Cap's orders as HoS to KoS friendly skeletons

**Comment by Savsj** [comment]
> Sure, but does that happen... uh... almost ever? I've been skeleton tons of times and have not once had that happen. The only time I've seen that happen is in the Armok video and that's a unique situation.

I have had that in 4 of 5 skeleton rounds ive played and all but a single round i have played as the ratking which is what we are proposing to turn these ghost roles into, if there was a middle between the two where they have the option of some antaggonism, but cannot be killed on sight like these roles can be it would be better.

>I see a missed opportunity to make the Cargo techs a neutral antag affiliated with a non-NT company (e.g. DonkCo).

This could be a really fun and interactive spawn for the cargo techs ghost role if it had a chance of spawning as like a representative of a neutral entity with different stuff on their shuttle and different outfits from time to time.

**Comment by thebadman4662** [comment]
But they can't be killed on sight, aside from syndies, which are obviously enemies of NT on top of not having crew rights. 

Other shuttle crews should get IDs, which they already got AFAIK but those are not visitor ones, rather station job probably so you should still direct them to HoP for icon change if they are not joining crew but just visit. 

Security simply have no reason to KOS them if they are not a threat, we can't exactly compare chef to space monster abomination, now, can we?

**Comment by SlamBamActionman** [comment]
> But they can't be killed on sight, aside from syndies, which are obviously enemies of NT on top of not having crew rights.
> 
> Other shuttle crews should get IDs, which they already got AFAIK but those are not visitor ones, rather station job probably so you should still direct them to HoP for icon change if they are not joining crew but just visit.
> 
> Security simply have no reason to KOS them if they are not a threat, we can't exactly compare chef to space monster abomination, now, can we?

I am more concerned with how *according to the* ***written*** *rules* these roles are differentiated from skeletons than what makes sense for them in-universe. We all agree that chefs should not be comparable to space monster abominations and should be treated with more hospitality from crew, the issue is if rules-wise they *are*.

File: Resources/Prototypes/Entities/Mobs/Player/humanoid.yml [hunk 1]
```diff
@@ -590,6 +590,8 @@
     - type: GhostTakeoverAvailable
     - type: Cluwne
 
+<<<<<<< HEAD
+=======
```
Comment (Lamrr) [inline Resources/Prototypes/Entities/Mobs/Player/humanoid.yml:594]: you probably didn't intend to leave this in here

**Comment by thebadman4662** [comment]
> > But they can't be killed on sight, aside from syndies, which are obviously enemies of NT on top of not having crew rights.
> > Other shuttle crews should get IDs, which they already got AFAIK but those are not visitor ones, rather station job probably so you should still direct them to HoP for icon change if they are not joining crew but just visit.
> > Security simply have no reason to KOS them if they are not a threat, we can't exactly compare chef to space monster abomination, now, can we?
> 
> I am more concerned with how _according to the_ _**written**_ _rules_ these roles are differentiated from skeletons than what makes sense for them in-universe. We all agree that chefs should not be comparable to space monster abominations and should be treated with more hospitality from crew, the issue is if rules-wise they _are_.

They are neutral so they can fight back and even kill if they want to, should they choose to do it. Of course it also gives them option to start something that could escalate into skeleton murderbone of sorts so admins would need to create yet another tier/type of antags if space chefs going postal should prove to be too much for them, which probably won't be a problem considering how skeletons fare.

**Comment by Djungelskog2** [comment]
> > > But they can't be killed on sight, aside from syndies, which are obviously enemies of NT on top of not having crew rights.
> > > Other shuttle crews should get IDs, which they already got AFAIK but those are not visitor ones, rather station job probably so you should still direct them to HoP for icon change if they are not joining crew but just visit.
> > > Security simply have no reason to KOS them if they are not a threat, we can't exactly compare chef to space monster abomination, now, can we?
> > 
> > 
> > I am more concerned with how _according to the_ _**written**_ _rules_ these roles are differentiated from skeletons than what makes sense for them in-universe. We all agree that chefs should not be comparable to space monster abominations and should be treated with more hospitality from crew, the issue is if rules-wise they _are_.
> 
> They are neutral so they can fight back and even kill if they want to, should they choose to do it. Of course it also gives them option to start something that could escalate into skeleton murderbone of sorts so admins would need to create yet another tier/type of antags if space chefs going postal should prove to be too much for them, which probably won't be a problem considering how skeletons fare.

the point is that we dont want or need another skeleton type role, the only reason skeletons arent too much of a pain is because A: they are a unique species, B: They dont have a role or identifier that would automatically make them crew or make people think they are crew and C: Skeletons are expected to behave however they want

**Comment by PotatOverlord** [comment]
There is no reason for the traveling chefs to be an antagonist of any kind; They are just chefs going around and cooking cool stuff. Allowing them to be neutral antags will invariably cause people to use them as an excuse to murderbone and poison the crew (Also an easy way to metagrudge someone). They can already defend themselves since they are under the same rules as crew, which allows self defense. Keeping them as friendly ghost roles will prevent NRP situations and keep the randomness and chaotic flair that the shuttle roles already bring. The syndicate shuttle being reworked was needed, but not in this way, for similar reasons, but I'll let other people argue for that because I'm lazy. Also the shuttle they use is an evac pod; it has no reverse thrust so maybe fix that if this goes through.

**Comment by K-Dynamic** [comment]
Yeah there should be some clarification on what type of neutral

A skeleton is definitely their own act of nature and can be as such, a travelling chef that's making food on a ship that they've probably got a mortgage on... yeah they're neutral to NT but shouldn't be able to roleplay destroying everyone

**Comment by Savsj** [comment]
I also think simply changing the syndie survivors to diet nukies is going to be an swift way to ruin a ghost role thats got a lot of good RP.

**Comment by GroggleG** [comment]
> I also think simply changing the syndie survivors to diet nukies is going to be an swift way to ruin a ghost role thats got a lot of good RP.

Most of the "good RP" ive seen so far is getting gunned down by sec (happened twice to me myself), I dont think something that is syndicate branded should be AT ALL crew rules NOR do I think it should be a true neutral. Though diet nukies is no fun at all. I know people dont like canning this stuff but I didnt like the cadets at all tbh.

**Comment by Savsj** [comment]
> > I also think simply changing the syndie survivors to diet nukies is going to be an swift way to ruin a ghost role thats got a lot of good RP.
> 
> Most of the "good RP" ive seen so far is getting gunned down by sec (happened twice to me myself), I dont think something that is syndicate branded should be AT ALL crew rules NOR do I think it should be a true neutral. Though diet nukies is no fun at all. I know people dont like canning this stuff but I didnt like the cadets at all tbh.

and thats a genuine problem of it being people not being properly informed of whats actually happening and what the role actually is, i have personally never seen the syndies gunned down but i play on mander. So YMMV but i think ultimately what this needs is a clarification vs a change.

**Comment by Dragonjspider** [comment]
In my opinion, the only shuttle that should have a neutral antag status is the syndie one. But even that has to have more restrictions than skeletons, since it doesn't seem reasonable for them to murder anyone or steal the armory. Like, for instance, they don't need to do anything antag related, but if a syndie decides to ask for help with objectives, I feel like they should have the right to accept them.

In fact, if you ask me, the skeleton is a chaotic neutral antag, as  they have no goals to speak of, they do what would be the most entertaining for them at the moment.
 
The biggest issue with the idea of making any of them neutral antags is the point that without an objective, you don't have any goals, which makes doing antag activities more appealing to do. I can already see a visiting chef deciding to kill a crewmember and making human food out of them if that happened.

Not to mention, like [IProduceWidgets](https://github.com/IProduceWidgets) said, an admin wouldn't have an easy way of identifying them as a neutral antag, since they're a human just like any other crewmember. Asking them to remember the names of up to 3 different people, all being neutral antags is too much, they have a lot more to pay attention to already...

> By making all of these roles able to have full agency to do whatever they want, they should be able to drive the round in whatever fashion they please, and eliminate the admin issues completely.

This wouldn't *remove* any issues admins have with the roles, it gives the roles more freedom to do things, which would muddy what's already a difficult role to gauge what the limit for them is. It goes from "well they just killed a guy, so that's a boink" to "well they just killed a guy, now I gotta double check whether or not they're doing it IC or if it's murderbone" 

The only way I can see this working at all is some kind of system that allowed you, as a neutral antag, to write a goal up, and send a request to something admins can see. So an admin might see their objective, and choose to allow it or not. But this gets into a whole other issue and is way out of scope...

**Comment by GroggleG** [comment]
> In my opinion, the only shuttle that should have a neutral antag status is the syndie one. But even that has to have more restrictions than skeletons, since it doesn't seem reasonable for them to murder anyone or steal the armory. Like, for instance, they don't need to do anything antag related, but if a syndie decides to ask for help with objectives, I feel like they should have the right to accept them.
> 
> In fact, if you ask me, the skeleton is a chaotic neutral antag, as they have no goals to speak of, they do what would be the most entertaining for them at the moment.
> 
> The biggest issue with the idea of making any of them neutral antags is the point that without an objective, you don't have any goals, which makes doing antag activities more appealing to do. I can already see a visiting chef deciding to kill a crewmember and making human food out of them if that happened.
> 
> Not to mention, like [IProduceWidgets](https://github.com/IProduceWidgets) said, an admin wouldn't have an easy way of identifying them as a neutral antag, since they're a human just like any other crewmember. Asking them to remember the names of up to 3 different people, all being neutral antags is too much, they have a lot more to pay attention to already...
> 
> > By making all of these roles able to have full agency to do whatever they want, they should be able to drive the round in whatever fashion they please, and eliminate the admin issues completely.
> 
> This wouldn't _remove_ any issues admins have with the roles, it gives the roles more freedom to do things, which would muddy what's already a difficult role to gauge what the limit for them is. It goes from "well they just killed a guy, so that's a boink" to "well they just killed a guy, now I gotta double check whether or not they're doing it IC or if it's murderbone"
> 
> The only way I can see this working at all is some kind of system that allowed you, as a neutral antag, to write a goal up, and send a request to something admins can see. So an admin might see their objective, and choose to allow it or not. But this gets into a whole other issue and is way out of scope...

I feel like that most of the issues with the whole syndicadets would be completely sidestepped if we had listening post made well, giving a more concrete way of identification and an actual goal. I just dont like the way that the syndicadets are implemented and should be given more time in the oven.

**Comment by XxSWAG-MASTER-OVERLORDxX** [comment]
I think with the clown shuttle, if you replaced the clown spiders with honk bots it would be a lot better

**Comment by TheShuEd** [comment]
map errors + merg conflict
i recreate PR later

**Comment by TheShuEd** [comment]
sorry wrong PR closed

**Comment by Brandon-Huu** [comment]
I vote to leave in clown shuttle.

**Comment by Flareguy** [comment]
Don't feel like working on this much lately + branch is cooked. I *might* get to this myself but would really appreciate if someone else got to the PR.

**Pull request closed without merging**